app/(authorized)/lines/(drawer)/menu.tsx

```
@@ -0,0 +1,41 @@
+import { API } from "@/services/api";
+import { routes } from "@/utils/utils";
+import { MaterialIcons } from "@expo/vector-icons";
+import { useMutation } from "@tanstack/react-query";
+import { useRouter } from "expo-router";
+import { Box } from "native-base";
+import { FC } from "react";
+import { ActivityIndicator, Pressable } from "react-native";
+
+type MenuPropsType = {};
+
+const Menu: FC<MenuPropsType> = ({}) => {
```

Biome

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)